### PR TITLE
Flowtype support

### PR DESCRIPTION
**Pull request by mukeshsoni**
Added support for basic types in flow - string, boolean, number, object, signature (custom types).

Have enabled the tests for flow type and also created the snapshots with minimal component example.

### CODE REVIEW

**Comment by mukeshsoni** [comment]
Added support for function generation which respects the signature of the function as defined in the flow type.

**Comment by typicode** [comment]
Thank you!

File: src/index.js [hunk 1]
```diff
@@ -67,26 +115,131 @@ function getFakePropType(prefix, prop, opts) {
...
+            //     "elements": [
+            //         {
+            //             "name": "Object"
+            //         }
+            //     ],
+            //     "raw": "Array<Object>"
+            // },
+            if (flowType.elements) {
+                return flowType.elements.map(prop => {
+                    return getFakeFlow('super', prop, opts)
```
Comment (typicode) [inline src/index.js]: why 'super' here?

**Comment by mukeshsoni** [comment]
That was supposed to be `prefix`. 

I had put 'super' since in our place we say 'super' when something works and we are happy :). Something like 'awesome!'.

**Comment by typicode** [comment]
interesting :) thanks for the explanation

**Pull request merged**